src/mon/OSDMonitor.cc

```diff
@@ -5437,10 +5437,12 @@ bool OSDMonitor::preprocess_command(MonOpRequestRef op)
     bool show_shadow = shadow == "--show-shadow";
     boost::scoped_ptr<Formatter> f(Formatter::create(format));
     if (f) {
+      f->open_object_section("crush_tree");
```

Comment:
@oshynsong my greatest concern is that this will break some tests/tools which extracts the `.nodes` array from the `ceph osd tree` output. as `ceph osd tree` is fairly a popular command in the Ceph world.